src/main/java/com/gluonhq/substrate/util/ios/Deploy.java

```diff
@@ -96,4 +103,49 @@ public static boolean install(String app) throws IOException, InterruptedExcepti
         }
         return false;
     }
+
+    public static void addDebugSymbolInfo(Path appPath, String appName) throws IOException, InterruptedException {
+        Path applicationPath = appPath.resolve(appName + ".app");
+        Path debugSymbolsPath = Path.of(applicationPath.toString() + ".dSYM");
+        if (Files.exists(debugSymbolsPath)) {
+            FileOps.deleteDirectory(debugSymbolsPath);
+        }
+        Path executablePath = applicationPath.resolve(appName);
+
+        Logger.logDebug("Generating debug symbol files...");
+        ProcessRunner runner = new ProcessRunner("xcrun", "dsymutil", "-o", debugSymbolsPath.toString(), executablePath.toString());
+        if (runner.runProcess("dsymutil") == 0) {
+            copyAppToProducts(debugSymbolsPath, executablePath, appName);
+        } else {
+            throw new RuntimeException("Error generating debug symbol files");
+        }
+    }
+
+    private static void copyAppToProducts(Path debugSymbolsPath, Path executablePath, String appName) throws IOException {
+        if (Files.exists(XCODE_PRODUCTS_PATH)) {
+            Files.walk(XCODE_PRODUCTS_PATH, 1)
+                    .filter(Objects::nonNull)
+                    .filter(path -> path.getFileName().toString().startsWith(appName))
+                    .forEach(path -> {
+                        try {
+                            Logger.logDebug("Removing older version: " + path.getFileName().toString());
+                            FileOps.deleteDirectory(path);
+                        } catch (IOException e) {
+                            Logger.logSevere("Error removing directory at " + path + ": " + e.getMessage());
+                        }
+                    });
+        }
+
+        String now = DateTimeFormatter.ofPattern("yyyyMMddHHmmss").format(LocalDateTime.now());
```

Comment:
Since `DateTimeFormatter` is thread safe, format `DateTimeFormatter.ofPattern("yyyyMMddHHmmss")` should be a constant

Reply:
Ok